lib/expand/builder.go

```diff
@@ -167,7 +181,6 @@ func (b *planBuilder) AddStartAgentPhase(plan *storage.OperationPlan) {
 				OpsCenterURL: fmt.Sprintf("https://%v", b.Peer),
 			},
 		},
-		Requires: []string{SystemPhase},
```

Comment:
Does this depend on another phase or does not have any dependency?

Reply:
It does not have any dependency anymore. Because we start the agent as a first phase now that we run pre-flight checks.